frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/ForgotPasswordPopup/ForgotPassword.js

```
@@ -0,0 +1,86 @@
+import React, { useState, useRef } from "react";
+import { Card, CardText, Modal } from "@selco/digit-ui-react-components";
+import { useTranslation } from "react-i18next";
+
+const ForgotPassword = ({ setPopup }) => {
+  const { t } = useTranslation();
+  const {contactPhone, contactEmail} = window?.globalConfigs?.getConfig("SUPPORT_CONTACTS");
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)